smalltalksrc/VMMakerTests/VMSpurScavengerTest.class.st

```diff
@@ -11,6 +11,27 @@ VMSpurScavengerTest >> assertPastSpaceIsEmpty [
 		equals: memory scavenger pastSpace start
 ]
 
+{ #category : #'as yet unclassified' }
+VMSpurScavengerTest >> fullNewSpace [
+
```

Comment:
How do you know this fills the new space?

Reply:
🌬️👆